samtranslator/schema/aws_serverless_connector.py

```diff
@@ -30,6 +30,11 @@ class Properties(BaseModel):
 class Resource(BaseModel):
     Type: Literal["AWS::Serverless::Connector"]
     Properties: Properties
+    DependsOn: Optional[PassThroughProp]
+    DeletionPolicy: Optional[PassThroughProp]
+    Metadata: Optional[PassThroughProp]
+    UpdateReplacePolicy: Optional[PassThroughProp]
+    Condition: Optional[PassThroughProp]
```

Comment:
Can we move these properties into a common class so that we don't have to duplicate these?